internal/executor/instance/pipe.go

```diff
@@ -82,12 +72,17 @@ func (pi *PipeInstance) Run(ctx context.Context, config *runconfig.RunConfig) (e
 			return
 		}
 
-		cleanupErr := agentVolume.Close(config.ContainerBackend)
+		containerBackend, creationError := config.GetContainerBackend()
```

Comment:
Same concern as in https://github.com/cirruslabs/cirrus-cli/pull/525/files#r892481899.

Reply:
Fixed in [59ee244](https://github.com/cirruslabs/cirrus-cli/pull/525/commits/59ee244f06b5d647d15d1e66ee1265a52147a4d2)